SwiftGit2Tests/Fixtures/Fixtures.swift

```diff
@@ -33,8 +33,14 @@ final class Fixtures {
 	
 	func setUp() {
 		NSFileManager.defaultManager().createDirectoryAtURL(directoryURL, withIntermediateDirectories: true, attributes: nil, error: nil)
-		
-		let bundle = NSBundle(identifier: "org.libgit2.SwiftGit2-OSXTests")!
+
+		#if os(OSX)
+			let platform = "OSX"
+		#else
+			let platform = "iOS"
+		#endif
```

Comment:
Can you remove the indentation from these preprocessor bits? And use normal indentation for the variables?